.github/workflows/CI.yml

```diff
@@ -3,26 +3,18 @@ on:
   pull_request:
     branches: [main]
   push:
-    branches: [main, test_CI]
+    branches: [main, test-ci]
 
 # TODO: create a game
 jobs:
   testbuild:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - name: "Checkout"
-        uses: actions/checkout@v2
-
-      - name: "Setup Python"
```

Comment:
Why remove this?

Reply:
I don't know what you mean. I upgraded `actions/checkout@v2` to `actions/checkout@v3`, and I removed `actions/setup-python` because I'm pretty sure that Python isn't needed to build the Docker image.